src/content-handlers/iiif/modules/uv-shared-module/BaseExtension.ts

```diff
@@ -1172,7 +1172,7 @@ export class BaseExtension<T extends BaseConfig> implements IExtension {
   }
 
   dispose(): void {
-    this.store?.destroy();
+    // pass
```

Comment:
[This discussion thread](https://github.com/pmndrs/zustand/discussions/2283) suggests that maybe `this.store?.setState(null);` would be a worthwhile replacement for the deprecated destroy call to help ensure appropriate garbage collection. What do you think?

Reply:
Thanks! I searched around, but mostly to see if `.destroy()` was every a zustand option.